flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/codegen/CodeGenerator.scala

```diff
@@ -1642,6 +1643,10 @@ abstract class CodeGenerator(
       s"""
          |$fieldTerm.open(new ${classOf[FunctionContext].getCanonicalName}($contextTerm));
        """.stripMargin
+    } else if (functionContextClass != null) {
+      s"""
+         |$fieldTerm.open(new ${functionContextClass.getCanonicalName}(parameters));
```

Comment:
parameters -> contextTerm ?

Reply:
contextTerm is an instance of RuntimeContext with we can not get when expression reducing, and parameters is an instance of Configuration.So we can't use contextTerm here

Reply:
You are right.